src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -559,6 +559,16 @@ def __init__(self, raw_response):
             fileTypeSummaries=[FileTypeSummary.for_bucket(bucket) for bucket in _sum['buckets']],
             cellCountSummaries=[OrganCellCountSummary.for_bucket(bucket) for bucket in _organ_group['buckets']])
 
+        # Un-translate values that had been translated with translate_fields() to handle Nones in Elasticsearch
+        # The path for each translation directly relates to the field used in transform_summary() for the aggregation
+        for summary in kwargs['cellCountSummaries']:
```

Comment:
This entire section looks brittle. If the dependent code is in this method, I'd be ok with it. Otherwise we may want to think about alternatives.

Reply:
Moved translation of values in `cellCountSummaries` into its constructor method `for_bucket()`.